src/components/toc.tsx

```diff
@@ -41,30 +41,87 @@ const TOCContainer = styled.div`
   top: 10px;
 `
 
+interface ItemProps {
+  isActive: boolean
+}
+
+const ListItem = styled.a<ItemProps>`
+  border-bottom: ${props => (props.isActive ? '1px solid' : '')};
```

Comment:
I'd do this to highlight the current item:
``` 
const ListItem = styled.a<ItemProps>`
  transition: all 0.2s ease-out;
  ${props => props.isActive ? 'text-color: theme.colors.gray900;' : ''}
`
```

Reply:
Ohhh smart, yeah i couldnt figure out how to animate the border without a hover action. Im not sure how well that plays with the `inlinecode`. Will check and see.

Reply:
Yeah I totally just did a github yolo coding so I only have a vague idea how this is going to look 😆

Reply:
> g to look 😆

Best kind of coding 👯‍♀️